dspace-api/src/test/resources/org/dspace/identifier/handle-forms.properties

```diff
@@ -0,0 +1,5 @@
+# Key is the identifier to be tested; value is "true" or "false":  should this
+# identifier be supported by the provider under test?
+
+http://dx.doi.org/10.14279/depositonce-5383 = false
+123456789/1 = true
```

Comment:
I had been thinking about "...only succeeds because...."  Actually, a proper unit test would set handle.prefix and handle.canonical.prefix to known values itself, before calling the target code.